ajax/libs/twbuttons/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "twbuttons",
+  "filename": "dist/twbuttons.css",
+  "description": "alexwolfe/Buttons for Twitter Bootstrap 3",
+  "license": "MIT",
+  "keywords": [
+    "buttons",
+    "less",
+    "css",
+    "css buttons",
+    "less buttons",
+    "bootstrap"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/onokumus/twbuttons.git"
+  },
+  "homepage": "https://onokumus.github.io/twbuttons/",
+  "author": "Osman Nuri Okumuş <onokumus@gmail.com> (https://github.com/onokumus)",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/onokumus/twbuttons.git",
+    "basePath": "dist"
```

Comment:
Yeah, It's about #10029 but it seems that the problem of #10029 is that it won't stop when the error been caught. I haven't started it yet...